core/domain/classifier_domain_test.py

```diff
@@ -16,148 +16,11 @@
 
 """Tests for classifier domain objects."""
 
-import datetime
-
 from core.domain import classifier_domain
 from core.tests import test_utils
 import utils
 
 
-class ClassifierTrainingJobDomainTests(test_utils.GenericTestBase):
```

Comment:
Keep this. It is part of the new framework.

Reply:
done.